src/m365/aad/commands/user/user-get.ts

```diff
@@ -50,7 +65,20 @@ class AadUserGetCommand extends GraphCommand {
 
     request
       .get(requestOptions)
-      .then((res: any): void => {
+      .then((res: any): Promise<any> => {
```

Comment:
We should avoid using `any` to avoid refactoring issues in the future. Since we know what data we expect from the API, we should use specific types instead.